### PR TITLE
 Add more trs

### LINKED ISSUE #114: internal message string shown to user
Observed in 'portion control' channel, item `7eb7f3a57f6449dbb9de90b7f10e3757`:

image

### LINKED ISSUE #119: perseus + vueintl localization errors during exam preview
### Observed behavior

Seeing errors during exam preview of an exam created using the portion control channel (`fb51dae6df7545af8455aa3a0c32048d`)

Looks like it occurred in the exercise "Large Decimal Rounding to Decimal Representations Practice"

image

### Expected behavior

Localization works as expected

### User-facing consequences

unknown

### Errors and logs

```
 [Vue Intl] Missing message: "perseusInternalMessages.a mixed number, like $1\ 3/4$" for locale: "en"
 [Vue Intl] Cannot format message: "perseusInternalMessages.a mixed number, like $1\ 3/4$", using message id as fallback.
 [Vue Intl] Missing message: "perseusInternalMessages.a multiple of pi, like $12\ \text{pi}$ or $2/3\ \text{pi}$" for locale: "en"
 [Vue Intl] Cannot format message: "perseusInternalMessages.a multiple of pi, like $12\ \text{pi}$ or $2/3\ \text{pi}$", using message id as fallback
```

### Steps to reproduce

* start to create an exam using portion control
* click 'preview'

### Context

Latest Kolibri Develop (0.8)

### PR DESCRIPTION
**Pull request by christianmemije**
Fixes #119
Fixes #114

### CODE REVIEW

File: kolibri_exercise_perseus_plugin/assets/src/translator.js [hunk 1]
```diff
@@ -3,30 +3,28 @@
 import { createTranslator } from 'kolibri.utils.i18n';
```
Comment (rtibbles) [inline kolibri_exercise_perseus_plugin/assets/src/translator.js]: In latest 0.7.x, I added an eslint disable to the top of this file. I forget why but it was necessary to enable building on 0.8.x. Could you rerun this using that?

Comment (christianmemije) [reply kolibri_exercise_perseus_plugin/assets/src/translator.js]: It's already on line 1:

image

Comment (christianmemije) [reply kolibri_exercise_perseus_plugin/assets/src/translator.js]: Or am I misunderstanding?

Comment (rtibbles) [reply kolibri_exercise_perseus_plugin/assets/src/translator.js]: No, I am just being blind, sorry.

**Comment by rtibbles** [comment]
This looks good to me, but going to hold off on merging until we have made a new release that goes into 0.8.x Kolibri.

**Pull request merged**